src/api/services/nodemailerService.ts

```diff
@@ -0,0 +1,47 @@
+import nodemailer from 'nodemailer';
+import { Address } from 'nodemailer/lib/mailer';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+import hbs from 'nodemailer-express-handlebars';
+
+import { EmailArgs, EmailData, UID2Sender } from './emailTypes';
+
+const transporter = nodemailer.createTransport({
+  host: 'localhost',
```

Comment:
does this have to change in non-local environemnt?

Reply:
no, this nodemailerService is only used by local env